packages/pcre/SOURCES/pcre0843.irix.patch

```diff
@@ -0,0 +1,90 @@
+diff -Naur pcre-8.43.orig/Makefile.am pcre-8.43/Makefile.am
+--- pcre-8.43.orig/Makefile.am	2016-03-02 13:56:27.000000000 +0000
++++ pcre-8.43/Makefile.am	2020-02-10 13:06:53.181543365 +0000
+@@ -442,46 +442,46 @@
+ 
+ endif # WITH_PCRE8
+ 
+-## There's a C++ library as well.
```

Comment:
Would be nice to know why this is being removed (and in fact in general, what is tested and what isn't(

Reply:
I'm unsure how to go about testing this test?

Reply:
Compiled grep from sgug-rse master + this pcre:

```
mips-sgi-irix6.5-gcc   -I/usr/sgug/include -O2 -g  -ldicl-0.1 -lintl -lc -llzma -lpthread -lgen  -o grep dfasearch.o grep.o kwsearch.o kwset.o pcresearch.o searchutils.o ../lib/libgreputils.a /usr/sgug/lib32/libintl.so -Wl,-rpath -Wl,/usr/sgug/lib32 ../lib/libgreputils.a /usr/sgug/lib32/libiconv.so -Wl,-rpath -Wl,/usr/sgug/lib32  -L/usr/sgug/lib32 -lpcre
dfasearch.o: In function `GEAcompile':
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:116: undefined reference to `re_set_syntax'
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:116: undefined reference to `re_set_syntax'
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:152: undefined reference to `re_compile_pattern'
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:152: undefined reference to `re_compile_pattern'
dfasearch.o: In function `EGexecute':
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:410: undefined reference to `re_search'
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:410: undefined reference to `re_search'
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:357: undefined reference to `re_search'
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:357: undefined reference to `re_search'
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:395: undefined reference to `re_match'
/usr/people/edodd/rpmbuild/BUILD/grep-3.3/src/dfasearch.c:395: undefined reference to `re_match'
collect2: error: ld returned 1 exit status
```